test/dashboardHandler.test.js

```diff
@@ -0,0 +1,369 @@
+            ],
+            "MetricName": "MTTF"
+          },
+          {
+            "Namespace": "Operations",
+            "Dimensions": [
+              {
+                "Name": "service",
+                "Value": "flaky-service"
+              }
+            ],
+            "MetricName": "MTBF"
+          }
+        ]
+      },
+      event: {
+        "account": "123456789012",
+        "region": "ap-southeast-2",
+        "detail": {},
+        "detail-type": "Scheduled Event",
+        "source": "aws.events",
+        "time": "2019-03-01T01:23:45Z",
+        "id": "cdc73f9d-aea9-11e3-9d5a-835b769c0d9c",
+        "resources": [
+          "arn:aws:events:ap-southeast-2:123456789012:rule/my-schedule"
+        ]
+      }
+    }, {
+      description: "sample dashboard - multiple service",
+      dashboard: "./sample-data/dashboard-multiple-sample.json",
+      uniqueServices: 2,
+      metrics: {
+        "Metrics": [
+          {
+            "Namespace": "Operations",
+            "Dimensions": [
+              {
+                "Name": "service",
+                "Value": "flaky-service"
+              }
+            ],
+            "MetricName": "MTTR"
+          },
+          {
+            "Namespace": "Operations",
+            "Dimensions": [
+              {
+                "Name": "service",
+                "Value": "flaky-service"
+              }
+            ],
+            "MetricName": "MTTF"
+          },
+          {
+            "Namespace": "Operations",
+            "Dimensions": [
+              {
+                "Name": "service",
+                "Value": "flaky-service"
+              }
+            ],
+            "MetricName": "MTBF"
+          },
+          {
+            "Namespace": "Operations",
+            "Dimensions": [
+              {
+                "Name": "service",
+                "Value": "stable-service"
+              }
+            ],
+            "MetricName": "MTTR"
+          },
+          {
+            "Namespace": "Operations",
+            "Dimensions": [
+              {
+                "Name": "service",
+                "Value": "stable-service"
+              }
+            ],
+            "MetricName": "MTTF"
+          },
+          {
+            "Namespace": "Operations",
+            "Dimensions": [
+              {
+                "Name": "service",
+                "Value": "stable-service"
+              }
+            ],
+            "MetricName": "MTBF"
+          }
+        ]
+      },
+      event: {
+        "account": "123456789012",
+        "region": "ap-southeast-2",
+        "detail": {},
+        "detail-type": "Scheduled Event",
+        "source": "aws.events",
+        "time": "2019-03-01T01:23:45Z",
+        "id": "cdc73f9d-aea9-11e3-9d5a-835b769c0d9c",
+        "resources": [
+          "arn:aws:events:ap-southeast-2:123456789012:rule/my-schedule"
+        ]
+      }
+    },
+    {
+      description: "sample dashboard - multiple service",
```

Comment:
The scenario name is the same as above - what's different about this one?

Reply:
Fixed it